packages/contracts-bedrock/contracts/L1/SystemConfig.sol

```diff
@@ -207,4 +257,69 @@ contract SystemConfig is OwnableUpgradeable, Semver {
             sstore(slot, _unsafeBlockSigner)
         }
     }
+
+    /**
+     * @notice A getter for the resource config. Ensures that the struct is
+     *         returned instead of a tuple.
+     */
+    function resourceConfig() external view returns (ResourceConfig memory) {
```

Comment:
Minor nitpick is to put the view functions together near the top. Not super opinionated on that.